epam-assets/theme/theme_loveship_dark.scss

```diff
@@ -412,15 +410,26 @@
         --uui-color-contrast-text: var(--uui-text-white);
     }
 
+    .uui-color-critical {
+        --uui-color-5: var(--uui-critical-5);
+        --uui-color-10: var(--uui-critical-10);
+        --uui-color-20: var(--uui-critical-20);
+        --uui-color-50: var(--uui-critical-50);
+        --uui-color-60: var(--uui-critical-60);
+        --uui-color-70: var(--uui-critical-40);
+        --uui-color-text: var(--uui-critical-50);
+        --uui-color-contrast-text: var(--uui-text-white);
+    }
+
     .uui-color-yellow {
         --uui-color-5: var(--yellow-5);
         --uui-color-10: var(--yellow-10);
         --uui-color-20: var(--yellow-20);
         --uui-color-50: var(--yellow-30);
         --uui-color-60: var(--yellow-40);
         --uui-color-70: var(--yellow-50);
-        --uui-color-text: var(--yellow-30);
-        --uui-color-contrast-text: var(--uui-text-white);
+        --uui-color-text: var(--yellow-50);
```

Comment:
Why you change from 30 to 50?  Below the same.

Reply:
because in our design color-text equals 50